packages/salesforcedx-vscode-core/src/commands/forceAuthLogout.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { nls } from '../messages';
+import {
+  EmptyParametersGatherer,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './commands';
+
+export class ForceAuthLogoutAll extends SfdxCommandletExecutor<{}> {
+  public static withoutShowingChannel(): ForceAuthLogoutAll {
```

Comment:
This is how I would instantiate a special command that doesn't show the output channel unnecessarily. I don't like this particular way of instantiating it. I would rather use some generic factory method but there seems to be issues with the type system of TypeScript to support that. I need to think about this  more but the issue is at https://github.com/Microsoft/TypeScript/issues/5863